pkg/apis/storage/types.go

```diff
@@ -374,6 +416,21 @@ const (
 // More modes may be added in the future.
 type VolumeLifecycleMode string
 
+// ServiceAccountToken contains parameters of a service account token.
+type ServiceAccountToken struct {
```

Comment:
Maybe I'm misreading, but basically `ServiceAccountToken` is not a service account token?

Reply:
yea, it contains parameters in TokenRequests. maybe i can rename it to `TokenRequest`. what do you think @liggitt

Reply:
renamed